### ISSUE TITLE
Nginx permission denied

### ISSUE THREAD

**Issue by swicaksono** [doccano/doccano#679]
After install the doccano using the docker-compose, I tried to access the website but it said "500 Internal Server Error - Nginx". I looked at the logs and it said,

```bash
nginx_1     | 2020/04/09 03:15:40 [crit] 6#6: *16 stat() "/var/www/html/index.html" failed (13: Permission denied), client: X.X.X.X, server: , request: "GET /favicon.ico HTTP/1.1", host: "X.X.X.X:8181", referrer: "http://X.X.X.X:8181/"
```

Oh yeah, I change the port access to 8181. How to solve this?

Your Environment
---------

*   Operating System: Ubuntu 16.04
*   Python Version Used: 3.7
*   When you install doccano: 04/09/2020 
*   How did you install doccano (Heroku button etc): docker-compose

**Comment by icoxfog417** [comment]
Please confirm the working of the original docker-compose before customizing the port. We want to clarify this is the original docker-compose issue or your customizing issue.

**Comment by swicaksono** [comment]
The story was all of my docker-compose is successfully deployed using this configuration [548](https://github.com/doccano/doccano/issues/548) but then my server got a bad sector, so all of the images and container was gone. but fortunately the doccano data can be backup. so I try run the docker-compose again with the old configuration:
- port  nginx listen I changed to 8181
- port expose in docker-compose-prod.yml I changed to 8181:8181

but I got an error like this,  "500 Internal Server Error - Nginx". surprisingly I can upload and send document using the doccano API but I can't access the doccano web. is it the docker-compose issues or the doccano itself? thank you for the response.
